freqtrade/rpc/telegram.py

```diff
@@ -418,7 +418,8 @@ def _prepare_entry_details(self, filled_orders: List, base_currency: str, is_ope
                 if prev_avg_price:
                     minus_on_entry = (cur_entry_average - prev_avg_price) / prev_avg_price
 
-                dur_entry = cur_entry_datetime - arrow.get(filled_orders[x-1]["order_filled_date"])
+                dur_entry = cur_entry_datetime - \
+                    arrow.get(filled_orders[x - 1]["order_filled_date"])
```

Comment:
please avoid unnecessary linebreaks (especially ones with `\` - which should not be used if possible). 
this line seems to be unchanged once it's reverted to one line.

Reply:
It's a [bug in pycodestyle](https://github.com/PyCQA/pycodestyle/issues/248#issuecomment-213658514)

Reply:
flake8 ignores E226 due to [this](https://github.com/PyCQA/pycodestyle/issues/248)

Reply:
hm - i noticed that these were ignored - but i assumed that's common for flake8?

Reply:
Good catch - on a separate note we should try to enable this at some point.

we don't configure `ignore = ` for flake8 - so the default applies - which apparently hides all warnings - so configuring the "seemingly default" throws a TON of warnings ...

Reply:
> Good catch

Using `Sublime Text`  with `anaconda` package which uses `autopep8`